### DISCUSSION TITLE
Firefox plugin version

### DISCUSSION THREAD

**Discussion by teslasystemsx** [Anarios/return-youtube-dislike#1050] [Q&A]
I see the latest version here is 3.0.0.16, but on the official addons page in Firefox it's 3.0.0.14. What's wrong with it?

**Comment by Anarios** [comment]
The current published release is [v4.0.5](https://github.com/Anarios/return-youtube-dislike/releases/tag/v4.0.5). The Firefox package has been submitted to Mozilla, but the official add-ons store update is still awaiting review; it requires Firefox 140 or newer. The store currently lists 3.0.0.18. Keeping this discussion open while the store update is pending; see also [#1288](https://github.com/Anarios/return-youtube-dislike/issues/1288).
